Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+# gem "rails"
+gem 'rubocop', '>= 1.0', '< 2.0'
```

Comment:
- [ ] To be able to run your tests, you need to ensure that you have the gem "rspec" listed here. Currently it is not listed and it is impossible to run the tests at the moment. KIndly include it ```gem 'rspec'```